source/manual/fall-back-to-mirror.html.md

```diff
@@ -1,125 +1,113 @@
 ---
 owner_slack: "#govuk-2ndline"
-title: Fall back to the static mirrors
+title: GOV.UK CDN static mirrors
 section: Deployment
 layout: manual_layout
 parent: "/manual.html"
 ---
 
-We maintain a static copy of most of the site, which gets used by the content delivery
-network (CDN) whenever origin (the application server) times out or serves an error
-response.
+GOV.UK uses the [Fastly content delivery network (CDN)](/manual/cdn.html) to handle most requests to [GOV.UK Origin](/manual/architecture-shallow-dive.html#a-user-visits-the-gov-uk-homepage). However, user requests will not get to Fastly in the following scenarios:
 
-This process is handled by our CDN config and is entirely transparent to us and
-our users. It happens multiple times a day, for lots of different reasons.
+- first and most commonly, user requests to GOV.UK may time out or return an error response
+- second, if Fastly goes down, user requests to Fastly will fail with a 5xx error code (the problem was caused by the server)
 
-This is why we refer to switching off Nginx on the origin cache machines as
-"falling back to the mirrors".
+When user requests to GOV.UK time out or return an error response, these requests instead go to a static copy of GOV.UK known as the GOV.UK mirror.
 
-## Hosting
+If Fastly has gone down, we [manually fall back to AWS CloudFront](/manual/fall-back-to-aws-cloudfront.html), the backup CDN. AWS then sends user requests to the GOV.UK mirror.
 
-GOV.UK is mirrored to two cloud providers:
+## GOV.UK mirror locations and access
 
-- Amazon S3: the static mirror is hosted in a bucket `govuk-<environment>-mirror` and the
-  content is retrieved by the CDN using an API.  This bucket is replicated to
-  `govuk-<environment>-mirror-replica` in another AWS region.
+The GOV.UK mirror is a copy of GOV.UK hosted in an [Amazon Web Services (AWS) S3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Welcome.html). The bucket contains copies of GOV.UK HTML files. The mirror is static, meaning dynamic pages such as search pages will not work.
 
-- Google GCS: the static mirror is hosted in a bucket `govuk-<environment>-mirror` and the
-  content is retrieved by the CDN using an API.
+There are 3 mirrors:
```

Comment:
Might be worth calling this out:

```suggestion
Whilst "the static mirror" implies only one mirror, it collectively refers to 3 mirrors:
```

Not happy with my wording, but hopefully you see what I'm getting at!